ui/packages/catalog-ui-search/src/main/webapp/react-component/container/histogram/index.spec.js

```diff
@@ -13,13 +13,12 @@
  *
  **/
 import * as React from 'react'
-import { expect } from 'chai'
 import { shallow } from 'enzyme'
 
 import { HistogramContainer } from '.'
 
 describe('<HistogramContainer />', () => {
   it('renders without error', () => {
```

Comment:
✏️ I believe you can get rid of `{` `}` now since there's just a function call now